src/pages/home/home.ts

```diff
@@ -349,7 +351,39 @@ export class HomePage {
           this.validDataFromClipboard = null;
           return;
         }
-        if (this.validDataFromClipboard.type === 'PayPro') {
+        if (this.validDataFromClipboard.type === 'InvoiceUri') {
+          const invoiceId: string = data.replace(
```

Comment:
"Stubs" PayProDetails from invoiceData to show the Invoice data in clipboard html page.